server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -213,10 +223,12 @@ private String getVideoId(String response) {
    * @param comments The array to be condensed.
    * @return A properly formatted String
    */
-  private String convertToString(ArrayList<String> comments) {
+  private String convertToString(ArrayList<Comment> comments) {
     StringBuilder res = new StringBuilder();
 
-    for (String comment : comments) {
+    for (Comment commentAndLikes : comments) {
+      String comment = commentAndLikes.text;
```

Comment:
nit: rename `commentAndLikes` to `comment`, and the current `comment` can be renamed to `commentText`